packages/formation/js/accordion.js

```diff
@@ -124,6 +124,22 @@ const addAccordionClickHandler = () => {
             if (!isElementInViewport(accordionButton)) {
               element.scrollIntoView();
             }
+
+            // Fire event for subscription by any consuming apps that need to
+            // handle click further (e.g., for analytics).
+            const accordionClickEvent = document.createEvent('Event');
+            accordionClickEvent.initEvent(
+              '@department-of-veterans-affairs/formation/accordion/button-clicked',
+              true,
+              true,
+            );
+            accordionClickEvent.detail = {
+              toggle:
+                accordionButton.getAttribute('aria-expanded') === 'true'
+                  ? 'expand'
+                  : 'collapse',
+            };
```

Comment:
I suppose this will read as `toggle: 'expand'` for an event fired from clicking an expanded accordion, but the input is in fact closing the accordion?

Reply:
@U-DON, yes, this event fires _after_ the widget-code updates the aria-expanded attribute.

I don't think we need any changes here, but do let me know if you think otherwise.